roles/openshift_preflight/common/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# check content available on all hosts
+- when:
+    - not containerized
+  block:
+
+    - name: determine if yum update will work
+      action: check_yum_update
+      register: r
+
+    - set_fact:
+        oo_preflight_check_results: "{{ oo_preflight_check_results + [r|combine({'_task': 'check content available on all hosts'})] }}"
```

Comment:
If the "language" allows for referencing variables and for setting variables (the `set_fact` module), I don't see why it would be an exception to reference a variable you are overriding?!

In pseudo-code:

```yaml
- set_fact:
    result: []

- set_fact:
    tmp: '{{ result }}'

- set_fact:
    result: '{{ tmp + [2] }}'
``` 

It would be a breaking change and probably a step backwards to require a `tmp` variable like that. That's not how Python works, not how Ansible works...